packages/react-vapor/src/components/table-hoc/TableHeaderWithSort.tsx

```diff
@@ -63,6 +64,14 @@ export class TableHeaderWithSort extends React.Component<
             'admin-sort-descending': this.props.sorted === false,
         });
 
+        if (this.props.isLoading) {
+            return (
+                <th id={this.props.id}>
+                    <div className="text-content-placeholder mod-small" />

```

Comment:
I think this could be a component, like `<LoadingPlaceholder mods=["mod-small"] />`

I can see myself wanting this in other projects 😄

Reply:
done!